website/docs/providers/proxy/forward_auth.mdx

```diff
@@ -23,8 +29,8 @@ This mode differs from the _Forward auth (single application)_ mode in the follo
 
 There are however also some downsides, mainly the fact that you **can't** restrict individual applications to different users.
 
-The only configuration difference between single application and domain level is the host you specify.
```

Comment:
I don't think this part should be removed as it still applies and is useful to people that can't use an existing config, maybe it needs some rewording though

Reply:
Oooof no i agree, I am not trying to remove it, was trying to move it up above, but am making a bit of a mess with this PR. Gimme a minute... ;-)

Reply:
I'd like to either go ahead and merge this PR as is, then keep working on the page, or close this PR
 and start over. Will for sure use your automated process for listing the linked docs.